_data/osdf_namespace_metadata/igwn-cit.yaml

```diff
@@ -19,6 +17,7 @@ display: true # Whether or not to display this namespace
 name: Caltech Gravitational Wave Data # Human readable name of the namespace
 namespace: 
   - "/igwn/cit"
+thirtyDayReads: null
+oneYearReads: null
```

Comment:
@turetske justin had null values, master had values. which should I keep?

Reply:
Take the `master` values -- Derek recently fixed some of his queries.

Reply:
okay, I'll update.